### ISSUE TITLE
no-member false positive with Django's ugettext_lazy

### ISSUE THREAD

**Issue by PCManticore** [PyCQA/pylint#1065]
Given the following snippet, we are getting a no-member false positive:

```
$ cat a.py
from django.utils.translation import ugettext_lazy as _
f = _("a")
f.format(2)
$ pylint a.py -rn
E: 3: Instance of '__proxy__' has no 'format' member (no-member)
```

**Comment by PCManticore** [comment]
Closing, this probably belong in pylint-django.
